mobify/platform-scaffold
# Change amp class prefix

Change `amp-` prefix class to `a-` class. The reason why we are making this change is because AMP built in are using `amp-` prefix so it will be best to avoid confusion between AMP built in classes and Mobify AMP classes. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-79

## Changes
- change `amp-` prefix to `a-`
- Remove `AmpLightbox` component, don't need right now. 
- Optimized some SVGs in the `/web` folder

## How to test-drive this PR
- Run `npm run dev`
- Check 
- Preview [Merlin's Potions PLP](http://localhost:3000/potions.html) and [Merlin's Potions PDP](http://localhost:3000/potions/eye-of-newt.html) and make sure that no bugs or errors found.
